src/commands/slack-notify-waiting-for-approval.yml

```diff
@@ -77,7 +77,7 @@ steps:
         name: Send Slack Message
         command: |
           set -eo pipefail
-          curl -X POST -H "Authorization: Bearer $PUSH_REMINDER_BOT_TOKEN" \
+          curl -X POST -H "Authorization: Bearer $SLACK_ACCESS_TOKEN" \
```

Comment:
Can you also update to use `curl --fail-with-body` so non 200 responses will generate a failure?

Reply:
Done.